hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+To adjust the current settings profile, you can assign to the current settings
+*instance* ``hypothesis.settings.default``.  This release makes it an error to
```

Comment:
Do we actually still allow this? I thought we didn't.

Reply:
Ooh, ambiguous sentence - that should be "assign to *attributes of* the current settings instance". I'll also expand the AttributeError message if you try to assign a new instance to point you to load_profile.

Reply:
Whoops, no, we don't!  I'll update the notes and error messages...